vaadin-portlet/pom.xml

```
@@ -31,7 +31,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>license-checker</artifactId>
-            <version>1.2.1</version>
+            <version>1.4.0</version>
```

```suggestion
            <version>1.4.1</version>
```
Should this update to the latest also?